README.md

```diff
@@ -122,7 +123,67 @@ If you want to deploy directly from the CLI, run `yarn vercel` and follow the st
 
 If you want to redeploy to the same production URL you can run `yarn vercel --prod`. If you omit the `--prod` flag it will deploy it to a preview/test URL.
 
-**Make sure your `packages/nextjs/scaffold.config.ts` file has the values you need.**
+**Make sure to check the values of your Scaffold Configuration before deploying your NextJS App.**
+
+### Scaffold App Configuration
+
+You can configure different settings for your dapp at `packages/nextjs/scaffold.config.ts`.
+
+```ts
+export type ScaffoldConfig = {
+  targetNetwork: chains.Chain;
+  pollingInterval: number;
+  alchemyApiKey: string;
+  walletConnectProjectId: string;
+  onlyLocalBurnerWallet: boolean;
+  walletAutoConnect: boolean;
+  // your dapp custom config, eg:
+  // tokenIcon : string;
+  // eventName : string;
+  // hideHeader : boolean;
```

Comment:
```suggestion

```

I feel only `tokenIcon` is sufficient maybe for example ?

Reply:
Yep, feels enough with 1 field! 🙌 
Fixed also the variable name to use `tokenIcon` in both the comment and the code snippet